Print_random_number_by_pushing_a_button/script.js

```diff
@@ -0,0 +1,30 @@
+/* let nodes = document.getElementsByTagName('button');
+for (let i = 0; i < nodes.length; i++) {
+   nodes[i].addEventListener('click', function Random(x) {
+      const max = 10;
+      return function () {
+         console.log('Element # ' + x + ' with random number - ' + Math.round(Math.random() * max));   // Работает. Но элементы повторяются.
+      }
+   }(i));
+} */
+
+let nodes = document.getElementsByTagName('button');
+for (let i = 0; i < nodes.length; i++) {
+   nodes[i].addEventListener('click', function Random(x) {
+      const max = 20;
+      return function () {
+         const uniqe = new Set();
+         while (uniqe.size !== max) {
```

Comment:
@Ovsiannikov78 This condition checks if unique size does not equal to max. Why do you need it?